src/firejail/fs_home.c

```diff
@@ -682,7 +686,7 @@ void fs_private_home_list(void) {
 		errExit("mounting tmpfs");
 	EUID_USER();
 
-	skel(homedir);
+	// skel(homedir);
```

Comment:
Why always disable the copying in this case?

Reply:
Because this was for debugging, and I forgot to commit that part. I'm really sorry, thank you for correcting me :) .